includes/class-wc-google-analytics.php

```diff
@@ -482,10 +482,42 @@ public function add_to_cart() {
 		$parameters['label']    = "'" . esc_js( $product->get_sku() ? __( 'ID:', 'woocommerce-google-analytics-integration' ) . ' ' . $product->get_sku() : "#" . $product->get_id() ) . "'";
 
 		if ( ! $this->disable_tracking( $this->ga_enhanced_ecommerce_tracking_enabled ) ) {
+			// Update SKU and attributes when variation selection changes
+			wc_enqueue_js(
+				'var wc_ga_integration_product_sku,
+					wc_ga_integration_variation_sku,
+					wc_ga_integration_variation_attributes;
+
+				wc_ga_integration_product_sku = "' . esc_js( $product->get_sku() ) . '";
+
+				$(document).on( "found_variation", "form.cart", function( e, variation ) {
+					wc_ga_integration_variation_attributes = variation.attributes;
+					if ( "undefined" !== typeof variation.sku && 0 !== variation.sku.length ) {
```

Comment:
:nail_care: 
Speaking of `.js` vs `.php` linting. In JS we avoid yoda conditions

Reply:
Good to know! Thanks 👍